src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/ReportedAdjudication.kt

```diff
@@ -48,7 +48,8 @@ data class ReportedAdjudication(
   var evidence: MutableList<ReportedEvidence>,
   @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
   @JoinColumn(name = "reported_adjudication_fk_id")
-  var witnesses: MutableList<ReportedWitness>
+  var witnesses: MutableList<ReportedWitness>,
+  var draftCreatedOn: LocalDateTime
```

Comment:
Small one - you could add a trailing comma at the end of this line. Same for `createReportedAdjudication`, `WithAValidDraftAdjudication` 1200 and 1300 and `reportedAdjudication`